integrations/wallets/ledger.md

```diff
@@ -73,6 +75,16 @@ In addition, in MetaMask, they must enable Ledger Live support. To do so, take t
  2. Navigate to "Advanced"
  3. Enable the "Use Ledger Live" feature
 
```

Comment:
maybe a header here could break things up... something like ### Connect Ledger to MetaMask 🤷‍♀️

Reply:
added!